[2024-08-06T21:55:46.272+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: load_parquet_to_bq_dag.dbt_run manual__2024-08-06T21:55:45.704136+00:00 [queued]>
[2024-08-06T21:55:46.277+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: load_parquet_to_bq_dag.dbt_run manual__2024-08-06T21:55:45.704136+00:00 [queued]>
[2024-08-06T21:55:46.277+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-08-06T21:55:46.278+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-08-06T21:55:46.278+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-08-06T21:55:46.285+0000] {taskinstance.py:1300} INFO - Executing <Task(DbtRunOperator): dbt_run> on 2024-08-06 21:55:45.704136+00:00
[2024-08-06T21:55:46.287+0000] {standard_task_runner.py:55} INFO - Started process 66 to run task
[2024-08-06T21:55:46.289+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'load_parquet_to_bq_dag', 'dbt_run', 'manual__2024-08-06T21:55:45.704136+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/load_dbt_dag.py', '--cfg-path', '/tmp/tmp41cyhui3']
[2024-08-06T21:55:46.289+0000] {standard_task_runner.py:83} INFO - Job 70: Subtask dbt_run
[2024-08-06T21:55:46.323+0000] {task_command.py:388} INFO - Running <TaskInstance: load_parquet_to_bq_dag.dbt_run manual__2024-08-06T21:55:45.704136+00:00 [running]> on host cf0df9e834a7
[2024-08-06T21:55:46.351+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=miguel
AIRFLOW_CTX_DAG_ID=load_parquet_to_bq_dag
AIRFLOW_CTX_TASK_ID=dbt_run
AIRFLOW_CTX_EXECUTION_DATE=2024-08-06T21:55:45.704136+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-06T21:55:45.704136+00:00
[2024-08-06T21:55:46.352+0000] {dbt_hook.py:117} INFO - dbt run --profiles-dir /opt/dbt_project --models load_parquet_to_test
[2024-08-06T21:55:46.356+0000] {dbt_hook.py:126} INFO - Output:
[2024-08-06T21:55:47.242+0000] {dbt_hook.py:130} INFO - [0m21:55:47  Running with dbt=1.8.4
[2024-08-06T21:55:47.848+0000] {dbt_hook.py:130} INFO - [0m21:55:47  Registered adapter: bigquery=1.8.2
[2024-08-06T21:55:47.918+0000] {dbt_hook.py:130} INFO - [0m21:55:47  Unable to do partial parsing because profile has changed
[2024-08-06T21:55:48.794+0000] {dbt_hook.py:130} INFO - [0m21:55:48  Found 1 model, 1 source, 473 macros
[2024-08-06T21:55:48.796+0000] {dbt_hook.py:130} INFO - [0m21:55:48
[2024-08-06T21:55:52.880+0000] {dbt_hook.py:130} INFO - [0m21:55:52  Concurrency: 1 threads (target='dev')
[2024-08-06T21:55:52.881+0000] {dbt_hook.py:130} INFO - [0m21:55:52
[2024-08-06T21:55:52.893+0000] {dbt_hook.py:130} INFO - [0m21:55:52  1 of 1 START sql table model prazos_project_dataset_prazos_project_dataset.test  [RUN]
[2024-08-06T21:55:54.079+0000] {dbt_hook.py:130} INFO - [0m21:55:54  BigQuery adapter: https://console.cloud.google.com/bigquery?project=prazos-project&j=bq:US:ae887101-e575-4390-93ac-a61f07180a76&page=queryresults
[2024-08-06T21:55:54.091+0000] {dbt_hook.py:130} INFO - [0m21:55:54  1 of 1 ERROR creating sql table model prazos_project_dataset_prazos_project_dataset.test  [[31mERROR[0m in 1.19s]
[2024-08-06T21:55:54.096+0000] {dbt_hook.py:130} INFO - [0m21:55:54
[2024-08-06T21:55:54.097+0000] {dbt_hook.py:130} INFO - [0m21:55:54  Finished running 1 table model in 0 hours 0 minutes and 5.30 seconds (5.30s).
[2024-08-06T21:55:54.135+0000] {dbt_hook.py:130} INFO - [0m21:55:54
[2024-08-06T21:55:54.135+0000] {dbt_hook.py:130} INFO - [0m21:55:54  [31mCompleted with 1 error and 0 warnings:[0m
[2024-08-06T21:55:54.135+0000] {dbt_hook.py:130} INFO - [0m21:55:54
[2024-08-06T21:55:54.136+0000] {dbt_hook.py:130} INFO - [0m21:55:54    Database Error in model load_parquet_to_test (models/load_parquet_to_test.sql)
[2024-08-06T21:55:54.136+0000] {dbt_hook.py:130} INFO -   Syntax error: Expected ")" but got ";" at [20:17]
[2024-08-06T21:55:54.136+0000] {dbt_hook.py:130} INFO -   compiled Code at target/run/prazos_project/models/load_parquet_to_test.sql
[2024-08-06T21:55:54.136+0000] {dbt_hook.py:130} INFO - [0m21:55:54
[2024-08-06T21:55:54.136+0000] {dbt_hook.py:130} INFO - [0m21:55:54  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
[2024-08-06T21:55:54.990+0000] {dbt_hook.py:132} INFO - Command exited with return code 1
[2024-08-06T21:55:54.997+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow_dbt/operators/dbt_operator.py", line 98, in execute
    self.create_hook().run_cli('run')
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow_dbt/hooks/dbt_hook.py", line 138, in run_cli
    raise AirflowException("dbt command failed")
airflow.exceptions.AirflowException: dbt command failed
[2024-08-06T21:55:55.000+0000] {taskinstance.py:1318} INFO - Marking task as UP_FOR_RETRY. dag_id=load_parquet_to_bq_dag, task_id=dbt_run, execution_date=20240806T215545, start_date=20240806T215546, end_date=20240806T215554
[2024-08-06T21:55:55.007+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 70 for task dbt_run (dbt command failed; 66)
[2024-08-06T21:55:55.044+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-08-06T21:55:55.052+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
